### PR TITLE
task/34215 create model test

### PR DESCRIPTION
**Pull request by yuyuvn**

### CODE REVIEW

File: spec/factories.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      Settings.question.answers.minimum.times do |i|
+        question.answers << FactoryGirl.build(:answer, question: question,
+          is_correct: i == 0)
+      end
+    end
+  end
+  
+  factory :category do
+    name FFaker::Lorem.sentence
+    max_time 100
```
Comment (phuongbk) [inline spec/factories.rb]: Cho 100 vào settings

File: spec/factories.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  factory :answer do
+    content FFaker::Lorem.sentence
+  end
+  
+  factory :question do
+    content FFaker::Lorem.sentence
+    before :create do |question|
+      Settings.question.answers.minimum.times do |i|
+        question.answers << FactoryGirl.build(:answer, question: question,
+          is_correct: i == 0)
```
Comment (phuongbk) [inline spec/factories.rb]: 0 to settings

File: spec/models/answer_spec.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+describe Answer, :type => :model do
```
Comment (phuongbk) [inline spec/models/answer_spec.rb]: Convention

File: spec/models/category_spec.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+describe Category, :type => :model do
```
Comment (phuongbk) [inline spec/models/category_spec.rb]: Convention

File: spec/models/exam_spec.rb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+describe Exam, :type => :model do
```
Comment (phuongbk) [inline spec/models/exam_spec.rb]: Convention

File: spec/factories.rb (hunk 1 above)
Comment (yuyuvn) [inline spec/factories.rb]: how about no?

**Pull request merged**